recorder.go

```diff
@@ -117,6 +118,9 @@ type Options struct {
 	// variable-length value types (strings, interface{}, etc).
 	MaxLogValueLen int `yaml:"max_log_value_len"`
 
+	// MaxLogsPerSpan limits the number of logs in a single span.
```

Comment:
No, we are setting `options.MaxLogsPerSpan` unconditionally after getting the `DefaultOptions()`. If we want that we can enclsoe that in a `if MaxLogsPerSpan != 0`